include/rellic/AST/GenerateAST.h

```diff
@@ -25,28 +26,15 @@ class GenerateAST : public llvm::AnalysisInfoMixin<GenerateAST> {
   friend llvm::AnalysisInfoMixin<GenerateAST>;
   static llvm::AnalysisKey Key;
 
+  constexpr static unsigned poison_idx = std::numeric_limits<unsigned>::max();
```

Comment:
What's the purpose of the `poison_idx`?

Reply:
It's the index of an expression that has not been found. See e.g. `GetReachingCond`